pulp_rpm/tests/functional/api/test_publish.py

```diff
@@ -439,30 +449,82 @@ def test_specified_package_checksum_type(self):
         for package, package_checksum_type in primary_checksum_types.items():
             self.assertEqual(package_checksum_type, "sha256")
```

Comment:
Here I would expect all to be sha512.
Let me know if I misunderstood something.

Reply:
Same